src/utils.js

```diff
@@ -34,4 +42,19 @@ export function fetchTopics(){
 export function handleTopicClick(topic, setTopic, setPage){
     setTopic(topic);
     setPage(1)
+}
+
+export function handleLikes(article_id, operator, setLikes){
+   
+    if (operator === 'minus'){
+        setLikes((current)=>{
+            return --current;
+        })
+    } else (setLikes((current)=>{
+        return ++current
+    }))
+    return axios.patch(`https://nc-news-qqh3.onrender.com/api/articles/${article_id}`, {
+            "inc_votes": operator === 'minus' ? -1 : 1
+        })
+
```

Comment:
Please use axios.create() in this file to have a base URL which can be user for each function.